Library/ENV/4.3/cc

```diff
@@ -1,6 +1,10 @@
-#!/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby -W0
+#!/bin/sh
+# Make sure this shim uses the same Ruby interpreter that is used by Homebrew.
+unset RUBYLIB
+unset RUBYOPT
+exec "$HOMEBREW_RUBY_PATH" -x "$0" "$@"
```

Comment:
> Maybe use same message in `config.rb` for scm wrapper?

I think that message would be misleading. It's not clear why `bin/brew` should be called instead. That's why I changed it slightly.

> But I think it can only be unset by build tools, right?

I would hope so. What do you suggest? Remove the check in the SCM wrapper, change it to use the message from the `cc` wrapper, keep it as-is, or something entirely different?